mac/sound.cpp

```diff
@@ -150,7 +150,7 @@ void CSound::GetAvailableInOutDevices()
             // in range
             if ( bIsInput_i && bIsOutput_j && ( lNumDevs < MAX_NUMBER_SOUND_CARDS ) )
             {
-                strDriverNames[lNumDevs] = "in: " + strDeviceName_i + "/out: " + strDeviceName_j;
+                strDriverNames[lNumDevs] = "in: " + strDeviceName_i + "/out: " + strDeviceName_j; //TODO: why can't this be translated?
```

Comment:
Is it a UI string?  If not, do not translate.

Reply:
See line 275, explicitly doing a string compare, for example.

Reply:
I must admit that I haven't looked at the code yet.